mirrord/layer/Cargo.toml

```diff
@@ -49,6 +49,7 @@ exec.workspace = true
 syscalls = { version = "0.6", features = ["full"] }
 null-terminated = "0.3"
 base64.workspace = true
+resolv-conf = "0.7"
```

Comment:
Should be moved to `[target.'cfg(target_os = "macos")'.dependencies]` section

Reply:
Correct